salt/orch/update.sls

```diff
@@ -57,9 +57,10 @@ update_modules:
   salt.function:
     - tgt: {{ master_id }}
     - name: cmd.run
-    - bg: True
     - arg:
-      - systemctl reboot
+      - sleep 15; systemctl reboot
```

Comment:
I think it's good enough, it's also possible to build the reboot time (now + 15 seconds) and provide it as an absolute value (`shutdown -r absolute`) instead of (`shutdown -r +mins`), but then you have funny clock skew problems and alike...

Keeping the LGTM, no changes required from my side.